Package/iOS/CMakeLists.txt

```diff
@@ -5,6 +5,14 @@ include(${CMAKE_CURRENT_SOURCE_DIR}/../../Apps/Playground/Playground/node_module
 set(PACKAGED_LIBS
     astc-codec
     astc
+	etc1
+	etc2
+	nvtt
+	squish
+	pvrtc
+	iqa
+	edtaa3
+	tinyexr
```

Comment:
I think these need to be added to `Modules/@babylonjs/react-native-iosandroid/react-native-babylon.podspec`, though I would have expected this to fail to link on iOS without that change. Is that not happening?

Reply:
No, it happens only when trying to link the app. I believed PG was built on the CI.